Makefile

```diff
@@ -53,6 +53,7 @@ cmd/util/util:
 .PHONY: update-core-contracts-version
 update-core-contracts-version:
 	./scripts/update-core-contracts.sh $(CC_VERSION)
+	make tidy
```

Comment:
is this one expected as part of this PR?

Reply:
I added this for convenience, so you don't have to call make tidy separately right after. I can put this in a separate PR, just to make this one a bit smaller.